Wordle.Api/Wordle.Api/Controllers/PersonController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using System.Numerics;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+[Route("Player")]
+public class PlayerController : ControllerBase
+{
+    private readonly PlayerService _PlayerService;
+    public PlayerController(PlayerService playerService)
+    {
+        _PlayerService = playerService;
+    }
+
+    [HttpGet]
+    public async Task<ActionResult<PlayerDto>> Get(Guid playerId)
+    {
+        Player? player = await _PlayerService.GetAsync(playerId);
+
+        if (player is not null)
+        {
+            return new PlayerDto(player);
+        }
+        return BadRequest();
+    }
+
+    [HttpGet("TopPlayers")]
+    public async Task<IEnumerable<PlayerDto>> GetTopPlayers()
+    {
+        return (await _PlayerService.GetTopPlayersAsync()).Select(player => new PlayerDto(player));
```

Comment:
I would suggest having your player server return a player dto, rather than doing it here.